package.json

```diff
@@ -8,7 +8,7 @@
     "John Gravois <jgravois@esri.com> (http://johngravois.com)"
   ],
   "dependencies": {
-    "esri-leaflet": "^2.0.0",
+    "esri-leaflet": "^3.0.2",
```

Comment:
Is this change necessary? It does not seem related to the main change in this PR.

Reply:
esri-leaflet had the same issue, I suppose you have to use an updated version otherwise you run into the same problem. See https://github.com/Esri/esri-leaflet/pull/1273

Reply:
another option: https://github.com/jgravois/esri-leaflet-gp/pull/46

Reply:
Any update about this? Currently running into the same issue
@gavinr you mentioned that this was fixed in esri-leaflet v3.0.2 (https://github.com/Esri/esri-leaflet/pull/1273#issuecomment-847204773) I'm guessing that's why the dependency to esri-leaflet is updated to v3.0.2

Reply:
Ran into the same issue, had to fork this library and force npm to resolve the package to my fork. Without fixing the version import and updating esri-leaflet to 3.0 the library cannot be used in projects built with webpack 5.